config/env_config.py

```diff
@@ -2,12 +2,20 @@
 from decouple import config
 from web3 import Web3
 from web3.middleware import geth_poa_middleware
+from helpers.enums import Network
 
 
 class EnvConfig:
     def __init__(self):
         self.test = config("TEST", "False").lower() in ["true", "1", "t", "y", "yes"]
         self.kube = config("KUBE", "True").lower() in ["true", "1", "t", "y", "yes"]
+        self.retroactive = config("RETROACTIVE", "True") in [
```

Comment:
Should this be flagged?  Verbiage looks like it's set to have this be retroactive distribution rather than current.